doc/CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# Contribute to the terraform plugin.
+
+So you want to be a terraform-libvirt-plugin hacker.
+
+# Where to start?
+
+Look at open issues, especially with label:
+
+- [junior job](https://github.com/dmacvicar/terraform-provider-libvirt/issues?q=is%3Aissue+is%3Aopen+label%3A%22junior+job%22), 
+- [help wanted](https://github.com/dmacvicar/terraform-provider-libvirt/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
+
+# Contribute
+
+1. [Build from source the plugin](https://github.com/dmacvicar/terraform-provider-libvirt#building-from-source)
```

Comment:
This sounds like Yoda. "Build the plugin from source" or something like that would be better.